neurodsp/filt/fir.py

```diff
@@ -155,7 +187,7 @@ def design_fir_filter(fs, pass_type, f_range, n_cycles=3, n_seconds=None):
     return filter_coefs
 
 
-def compute_filter_length(fs, pass_type, f_lo, f_hi, n_cycles=None, n_seconds=None):
+def compute_filter_length(fs, pass_type, f_lo, f_hi, n_cycles=3, n_seconds=None):
```

Comment:
If `n_cycles` defaults to None, while `n_seconds` here also defaults to None, then the default behavior will be to raise a ValueError. I don't think is intended?

Reply:
Yeh, at least one of them does need to be defined, but I don't think we want to assume n_cycles is 3 (even though it's default elsewhere). If you make a custom filter, and call this without n_cycles or n_seconds, it will use the default of 3, which could be wrong, and then silently give the wrong answer.

Reply:
I'll change this back to `n_cycles=None`. This will require either n_cycles or n_seconds to be defined. Maybe one (or both) of these should be positional arguments instead optional, keyworded arguments.